draft-ietf-ohai-ohttp.md

```diff
@@ -249,7 +249,8 @@ Oblivious HTTP is more costly than a direct connection to a server.  Some costs,
 like those involved with connection setup, can be amortized, but there are
 several ways in which Oblivious HTTP is more expensive than a direct request:
 
-* Each request requires at least two regular HTTP requests, which adds latency.
+* Each request requires at least two regular HTTP requests, which adds
+  processing latency.
```

Comment:
```suggestion
* Each request requires at least two regular HTTP requests, which could
  increase latency.
```

Reply:
I would also be fine with this proposal.

Reply:
Yes, I think this is a way out of it too =)